src/pages/public_registration/action_accordions/DuplicateWarningModal.tsx

```diff
@@ -0,0 +1,92 @@
+import OpenInNewOutlinedIcon from '@mui/icons-material/OpenInNewOutlined';
+import { Box, Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Modal } from '../../../components/Modal';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getRegistrationLandingPagePath } from '../../../utils/urlPaths';
+
+interface DuplicateWarningModalProps {
+  isOpen: boolean;
+  toggleModal: () => void;
+  onConfirmNotDuplicate: () => void;
+  duplicateId: string | undefined;
+}
+
+export const DuplicateWarningModal = ({
+  isOpen,
+  toggleModal,
+  duplicateId,
+  onConfirmNotDuplicate,
+}: DuplicateWarningModalProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Modal
+      open={isOpen}
+      onClose={() => {
+        toggleModal();
+      }}
+      maxWidth="xs"
+      fullWidth={true}
+      headingText={t('registration.public_page.duplicate_warning_modal.headline')}
+      dataTestId={dataTestId.registrationLandingPage.duplicateRegistrationModal.duplicationModal}>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+        <Typography>{t('registration.public_page.duplicate_warning_modal.publication_already_exists')}</Typography>
+        {duplicateId && (
+          <Box>
+            <Typography sx={{ display: 'inline' }}>
+              {`${t('registration.public_page.duplicate_warning_modal.dont_create_duplicates')} ${t('registration.public_page.duplicate_warning_modal.check_if')}`}
+            </Typography>
+            <Link
+              target="_blank"
+              data-testid={dataTestId.registrationLandingPage.duplicateRegistrationModal.duplicateRegistrationLink}
+              to={getRegistrationLandingPagePath(duplicateId)}>
+              <Box sx={{ gap: '0.25rem', display: 'inline-flex', mr: '0.25rem' }}>
+                <Typography
+                  sx={{ display: 'inline', textDecoration: 'underline', cursor: 'pointer', color: 'primary.light' }}>
+                  {t('registration.public_page.duplicate_warning_modal.this_publication')}
+                </Typography>
+                <OpenInNewOutlinedIcon
```

Comment:
Tidligere mener jeg det har det vært snakk om at dette ikonet skal brukes for å indikere at man forlater NVA, mener det ble sagt ifbm oppdatering av footer for en tid tilbake. Så jeg ville antatt denne ikke skal være her, men om noen sier den skal være med er det greit for meg :p 
Samme gjelder linken i høyremeny ser jeg

Edit: Ser det er sånn i skissene også, selv om det ikke er i tråd med det vi har snakket om før da. Kan godt bare la det være sånn, så får vi ta en diskusjon om hvordan lenker skal se ut når designer er tilbake. Ser også at lenkene her er blå i skissene, men det er også noe vi aktivt har gått bort fra andre plasser før, ved å overstyre lenke-fargen i `mainTheme`, så flere ting vi må diskutere her. Akkurat nå endrer vi default HTML sine blå lenke-farger til vår egen i theme, og så overstyrer vi enda en gang til blå noen plasser, så noe opprydning må vi hvertfall få gjort 😅

Reply:
Ser at vi mikser en del fra før også, hvor vi bl.a. har NVA-linker i footer, mens på personvern-siden har vi native `<a>`-linker. Mulig å bruke bare `Link` fra react-router hadde vært best om vi ønsker at de skal være blå 🤔